app/views/artifacts/show.html.erb

```diff
@@ -3,7 +3,7 @@
 <p>Material: <%= @artifact.material %></p>
 <p>Date Created: <%= @artifact.year_created %></p>
 <p>Total Pieces: <%= @artifact.total_pieces %></p>
-<p>On Loan from Another Museum? <%= @artifact.on_loan %></p>
+<p>On Loan from Another Museum: <%= @artifact.on_loan %></p>
```

Comment:
I have changed this multiple times, not sure why it continues to revert back to old code??